### DISCUSSION TITLE
More than 50 minutes looking for a game

### DISCUSSION THREAD

**Discussion by PeraFeliz** [ValveSoftware/Dota2-Gameplay#22590] [Q&A]
Currently, searching for ranked matches is taking anywhere from 50 minutes to an hour or even longer, with no success in finding a match. This issue significantly affects my gaming experience, as I am unable to participate in ranked matches on a regular basis.

It is worth mentioning that before I acquired the Dota Plus subscription, I was able to find matches quickly and had no issues in searching. I don't know if this can be related, but since then I started experiencing these difficulties. I have checked my game settings and the quality of my internet connection, and everything seems to be in order; however, the problem persists. I have also tried searching for matches at different times to see if it improves, but the problem persists.

I would greatly appreciate if you could provide me with some recommendations.

Thank you in advance for your attention.
![PROBLEMA](https://github.com/user-attachments/assets/929e0f3d-b977-4eaa-ac5f-9b8d986d23b8)

**Comment by minami093004** [comment]
man dota 2 should fix this kind of problem
